src/questions/makerdao.yaml

```diff
@@ -0,0 +1,245 @@
+- uuid: cb274dad-bb8d-484a-83a5-486530ddff69
+  type: SingleChoice
+  content: What are the two tokens in MakerDAO's two token system?
+  hint: NoHint
+  explanation: MKR and DAI
+  answerKeys:
+    - A
+  difficultyLevel: "low"
+  choices:
+    - content: MKR and DAI
+      key: A
+      
+    - content: DAI and SAI
+      key: B
+      
+    - content: DAI and VAI
+      key: C
+      
+    - content: MKR and DAO
+      key: D
+      
+- uuid: 567a6a91-b11f-4823-908f-a3a7903ce614
+  type: SingleChoice
+  content: How is DAI borrowed?
+  hint: NoHint
+  explanation: DAI is borrowed by opening an overcollateralized debt position by creating a maker vault
+  answerKeys:
+    - D
+  difficultyLevel: "Medium"
+  choices:
+    - content: DAI can no longer be borrowed in MakerDAO
+      key: A
+      
+    - content: By buying it at a 1:1 ratio with USD
+      key: B
+      
+    - content: By requesting MakerDAO officials
+      key: C
+      
+    - content: By opening an overcollateralized borrow position in a Maker vault
+      key: D
+      
+- uuid: 27b70dc5-b43e-46f9-a311-2d545ad3794c
+  type: MultipleChoice
+  content: All Maker vaults have the same set of risk parameters (for example, dust)
```

Comment:
Expand a bit more on risk parameter. Is it used to determine how much DAI can a user borrow?